core/hybrisadaptor.cpp

```diff
@@ -422,14 +474,54 @@ HybrisManager *HybrisManager::instance()
 
 #ifdef USE_BINDER
 
+GBinderLocalReply *HybrisManager::sensorCallbackHandler(
+    GBinderLocalObject* obj,
+    GBinderRemoteRequest* req,
+    guint code,
+    guint flags,
+    int* status,
+    void* user_data)
+{
+    (void)flags;
+    (void)obj;
+    (void)user_data;
+    sensordLogD() << "sensorCallbackHandler";
+    const char *iface = gbinder_remote_request_interface(req);
+    if (iface && (!strcmp(iface, SENSOR_BINDER_SERVICE_IFACE_2_0) ||
+                  !strcmp(iface, SENSOR_BINDER_SERVICE_IFACE_2_1)
+                  )) {
+        switch (code) {
+        case DYNAMIC_SENSORS_CONNECTED_2_0:
+        case DYNAMIC_SENSORS_CONNECTED_2_1: {
+            sensordLogD() << "Dynamic sensor connected";
+            break;
+        }
+        case DYNAMIC_SENSORS_DISCONNECTED_2_0: {
+            sensordLogD() << "Dynamic sensor disconnected";
+            break;
+        }
```

Comment:
nit: These case x: { } brackets ... are not needed, looks odd.

Reply:
Those seem to be remnant from some code from another project I made and used as example. I will remove.